packages/headless/src/features/facets/facet-set/facet-set-selectors.ts

```diff
@@ -1,12 +1,12 @@
 import {
-  OldProductListingSection,
+  ProductListingSection,
   SearchSection,
 } from '../../../state/state-sections';
 import {FacetSection} from '../../../state/state-sections';
 import {AnyFacetResponse} from '../generic/interfaces/generic-facet-response';
 import {FacetResponse, FacetValue} from './interfaces/response';
 
-export type FacetResponseSection = SearchSection | OldProductListingSection;
+export type FacetResponseSection = SearchSection | ProductListingSection;
```

Comment:
This can just be `SearchSection`. In fact, we may not need this type at all anymore.